### PR TITLE
Docs: rename scrape_native_histograms setting based on prometheus update

### PR DESCRIPTION
**Pull request by tacole02**
#### What this PR does

This PR updates the `scrape_native_histograms` setting to `always_scrape_native_histograms` at https://grafana.com/docs/mimir/latest/send/native-histograms/. This update is based on a [change in Prometheus version 3.0](https://github.com/prometheus/prometheus/pull/15178).

#### Which issue(s) this PR fixes or relates to

Fixes https://github.com/grafana/mimir/issues/9695

#### Checklist

- [ ] Tests updated.
- [X] Documentation added.
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`.
- [ ] [`about-versioning.md`](https://github.com/grafana/mimir/blob/main/docs/sources/mimir/configure/about-versioning.md) updated with experimental features.

### CODE REVIEW

**Comment by tacole02** [comment]
Hi @krajorama , please approve and merge this PR if the change looks good to you! Thanks!

**Review by 56quarters — APPROVED** [review]
LGTM

**Comment by tacole02** [comment]
Thanks so much, @56quarters ! Do you mind merging this PR? I don't have merge rights for this repo.

**Pull request merged**
